test/format/format_config_def.c

```diff
@@ -57,8 +57,8 @@ CONFIG configuration_list[] = {
   {"btree.memory_page_max", "maximum cache page size",
     C_TABLE, 1, 10, 128, V_TABLE_BTREE_MEMORY_PAGE_MAX},
 
-  {"btree.prefix_len", "common key prefix", C_TABLE | C_TYPE_ROW | C_ZERO_NOTSET,
-    PREFIX_LEN_CONFIG_MIN, PREFIX_LEN_CONFIG_MAX, PREFIX_LEN_CONFIG_MAX, V_TABLE_BTREE_PREFIX_LEN},
+  {"btree.prefix_len", "common key prefix",
+    C_TABLE | C_TYPE_ROW | C_ZERO_NOTSET, PREFIX_LEN_CONFIG_MIN, PREFIX_LEN_CONFIG_MAX, PREFIX_LEN_CONFIG_MAX, V_TABLE_BTREE_PREFIX_LEN},
```

Comment:
It looks like there are a bunch of changes here that probably just came from `clang-format`. Or am I missing something?

Reply:
Yes, different formatting styles.  I've opened WT-10793 to "fix" this.

Reply:
Thanks!